proposals/evaluation/uc-0-effective-acr.md

```diff
@@ -184,13 +192,46 @@ GRAPH </.default.acr> {
 }
 ```
 
-This should be read as giving the triples in the `</.default.acr>` graph, and specifying that no triples exist in the `</foo/bar/baz/x.acr>`.
-With a slight adjustment to the WAC spec, this could still count as there being nothing other than the default, which would therefore still be active.
+This should be read as 
+ * specifying that no triples exist in`</foo/bar/baz/x.acr>` 
+ * giving the triples in the `</.default.acr>` graph
 
-Note: NTrig has already been proposed in [issue 210: add :imports relation](https://github.com/solid/authorization-panel/issues/210).
+With a slight adjustment to the WAC spec, this could still count as there being nothing other than the default, which would therefore still be active as default for the given resource.
 
+#### WAC+:imports
 
+Another way to allow default reasoning is to use `:imports` as proposed in [issue 210: add :imports relation](https://github.com/solid/authorization-panel/issues/210). 
 
+[Reactive Solid](https://github.com/co-operating-systems/Reactive-SoLiD) has every resource come with an ACR with content. But the server has ACRs for newly created resources explictly `:import` the parent, so that `</foo/bar/baz/x.acr>` returns
```

Comment:
```suggestion
[Reactive Solid](https://github.com/co-operating-systems/Reactive-SoLiD) has every resource come with an ACR with content. But the server has ACRs for newly created resources explictly `:import` the parent, so that `</foo/bar/baz/x.acr>` returns —
```